samples/client/petstore/typescript-axios/tests/with-complex-headers/pom.xml

```diff
@@ -0,0 +1,59 @@
+<project>
```

Comment:
this has to be registered similar to https://github.com/OpenAPITools/openapi-generator/blob/57cfff195080e4f983d849d41037584132f326ee/CI/circle_parallel.sh#L101